### DISCUSSION TITLE
How to animate a single image?

### DISCUSSION THREAD

**Discussion by YRlin-12** [showlab/MotionDirector#27] [Q&A] [answered]
Great work!
I am wondering how to reproduce the Row4 of Figure 2 in the main paper.
Could you please give me some advice?
Do we need inversion latents of the reference video in this case?

**Comment by ruizhaocv** [comment]
Thanks for your interest. The code will be released.

**Comment by YRlin-12** [comment]
> Thanks for your interest. The code will be released.

Please tell us do you have any plans to release the code? 
Thank you!

**Comment by ruizhaocv** [comment]
Yes. We are cleaning the code and preparing for appearance customization. We plan to release the code next week. Please stay tuned.

**Comment by ruizhaocv** [comment, marked as answer]
Hi. The code has been released. Please refer to the [link](https://github.com/showlab/MotionDirector?tab=readme-ov-file#motiondirector-for-image-animation-) for instructions.
